lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -505,6 +505,7 @@ def links_for(serializer)
       # prs:
       #   https://github.com/rails-api/active_model_serializers/pull/1041
       def pagination_links_for(serializer)
+        return {} unless instance_options.key?(:serialization_context)
```

Comment:
Why not put this test [there](https://github.com/bf4/active_model_serializers/blob/9b77f0af2928051e81984bce96dc365e7adb4cda/lib/active_model_serializers/adapter/json_api.rb#L129)?